backend/regalloc/regalloc_gi.ml

```diff
@@ -109,15 +109,28 @@ let make_hardware_registers_and_prio_queue (cfg_with_infos : Cfg_with_infos.t) :
 (* CR xclerc for xclerc: try to find a reasonable threshold. *)
 let max_rounds = 32
 
+let max_temp_multiplier = 10
+
 (* CR xclerc for xclerc: the `round` parameter is temporary; this is an hybrid
    version of "greedy" using the `rewrite` function from IRC when it needs to
    spill. *)
-let rec main : round:int -> flat:bool -> State.t -> Cfg_with_infos.t -> unit =
- fun ~round ~flat state cfg_with_infos ->
+let rec main :
+    round:int ->
+    flat:bool ->
+    temporaries:int ->
```

Comment:
I think it might be better to
have `temporaries` stored
in `State.t`.

Reply:
Done